src/org/omegat/core/data/RealProject.java

```diff
@@ -1814,17 +1815,24 @@ static class AlignFilesCallback implements IAlignCallback {
             this.config = props;
         }
 
-        Map<String, TMXEntry> data = new TreeMap<>();
-        private ProjectProperties config;
+        Map<EntryKey, TMXEntry> data = new TreeMap<>();
```

Comment:
I think we could use ITMXEntry here, instead of TMXEntry. Sure this interface did not exist when previous version was written. But this has some advantages, see my comment on line 1860/1868